packages/api/README.md

```diff
@@ -2,116 +2,33 @@
 
 The nft.storage public API.
 
-## Getting Started
-
-### Local env vars
+## Usage
 
-Follow instructions in the root `README.md`.
+Make sure you already did step 1 and 2 from these [instructions](/#getting-started).
```

Comment:
```suggestion
Make sure you already did step 1 and 2 from the [root instructions](/#getting-started).
```
Let's be more clear to avoid people to need to navigate if they know they just did it